README.md

```diff
@@ -1,19 +1,78 @@
 # HDL Reference Designs
 
-[Analog Devices Inc.] HDL libraries and projects
+[Analog Devices Inc.](http://www.analog.com/en/index.html) HDL libraries and projects.
 
-### Branches
+## Getting started
 
-Each release has its own branch and master always synced with the latest release. 
-To find out more information about the latest release please check the release notes.
-Every branch, which has **dev** in its name, is a development branch and should handle it accordingly. 
+These projects currently supporting several different [FPGA development boards](https://github.com/analogdevicesinc/hdl/tree/master/projects) and [FMC boards](https://github.com/analogdevicesinc/hdl/tree/master/projects/common). 
+First make sure that the current state of the repository support's your hardware.
```

Comment:
I think the statement is not necessary.

Reply:
I think you have to have a clear statement that describes the purpose of the repo. I'm not saying that this statement is the best though.